docs/docs/1.2.0/documentation/dps-and-data-model.md

```diff
@@ -371,22 +379,17 @@ You can provide your own public address and port using the following annotations
 In Universal mode the dataplane resource should be deployed as follows:
 
 ```yaml
-type: Dataplane
-mesh: default
+type: ZoneIngress
 name: dp-ingress
 networking:
   address: 192.168.0.1
-  ingress:
-    publicAddress: 10.0.0.1
-    publicPort: 10000 
-  inbound:
-  - port: 10001
-    tags:
-      kuma.io/service: ingress
+  port: 10001
+  advertisedAddress: 10.0.0.1
+  advertisedPort: 10000
 ```
 ::::
 
-Ingress deployment can be scaled horizontally. Many instances can have the same public address and port because they can be put behind one load balancer.
+`Zone Ingress` deployment can be scaled horizontally. Many instances can have the same advertised address and advertised port because they can be put behind one load balancer.
```

Comment:
```suggestion
A `ZoneIngress` deployment can be scaled horizontally. Many instances can have the same advertised address and advertised port because they can be put behind one load balancer.
```